server/src/Lib.hs

```diff
@@ -147,42 +152,65 @@ buildTriplet appDB trip@(BinaryTriplet bin pkg sys) = withApplicationDB appDB $
     -- TODO is this the right way to do thread safety? or should we take more care on the application side?
     bump conn = liftIO $ execute conn "UPDATE binaries SET hits = hits + 1 WHERE binary = ? AND package = ? AND platform = ?" (bin, pkg, show sys)
 
+-- | Write the data from the input 'Handle' to the output 'Handle' and collect
+-- the read bytes into a lazy bytestring.
+tee' :: Handle -> Handle -> IO Lazy.ByteString
+tee' input output = do
```

Comment:
Why is this ticked?

Reply:
It's ticked because the platonic `tee` function would take three handles and, reading from the first and writing to the other two and return `IO ()`. The tick here signifies that this is a modification on a more canonical `tee`. If you have a better name for `tee'` that'd be nice, but I don't think it should be `tee`.